geps/gep-713/index.md

```diff
@@ -1043,6 +683,9 @@ Support: Provisional
 
 This solution is IN PROGRESS and so is not binding yet.
 
+However, a version of this proposal is now included in the Direct Policy
+Attachment GEP.
+
 This solution requires definition in a GEP of its own to become binding.
```

Comment:
It would be good to start the issue for the PolicyAncestor GEP and include a reference to it here.